crepr/crepr.py

```diff
@@ -335,10 +358,11 @@ def add(
     files: Annotated[list[pathlib.Path], file_arg],
     kwarg_splat: Annotated[str, splat_option] = "{}",
     diff: Annotated[Optional[bool], diff_inline_option] = None,  # noqa: UP007
+    ignore_existing: Annotated[bool | None, ignore_existing_option] = None,
```

Comment:
Does it work with
```suggestion
    ignore_existing: Annotated[bool, ignore_existing_option] = False,
```
I don't think we need the None here.

Reply:
With the suggested changes ruff will start complaining, I can add `#noqa: FBT002` to supress that. 
Or is there anything else that you think will be better?

Reply:
`#noqa: FBT002` is fine in this case